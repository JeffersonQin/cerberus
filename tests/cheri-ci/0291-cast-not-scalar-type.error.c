struct s { int x; };
int main()
{
  int x;
  (struct s) x;
}
