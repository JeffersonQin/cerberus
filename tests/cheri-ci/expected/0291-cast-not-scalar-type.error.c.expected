cheri-ci/0291-cast-not-scalar-type.error.c:5:14: error: constraint violation: used type 'struct s' where scalar type is required
  (struct s) x;
  ^          ~ 
ยง6.5.4#2: 
2   Unless the type name specifies a void type, the type name shall specify atomic, qualified,
    or unqualified scalar type, and the operand shall have scalar type.
